components/Workspaces/WorkspaceLayout.tsx

```diff
@@ -60,8 +69,26 @@ export const WorkspaceLayout = ({ workspaceId, children }: WorkspaceLayoutProps)
           )}
         </ClientOnly>
       </div>
-      <div className="flex flex-col items-center grow pt-8 md:pt-14 lg:pt-20">
+      <div className="relative flex flex-col items-center grow pt-8 md:pt-14 lg:pt-20">
```

Comment:
not what I mentioned about using React portals the other day, but I envisioned it more passing the component and use the `bannerProps` prop (probably rename it to `banner`) like a slot.

```jsx
<WorkspaceLayout workspaceId={workspaceId} banner={<p>Hello</p>}>
...
</WorkspaceLayout>
```

and in the WorkspaceLayout component.

have e.g. `banner` props is optional.

```jsx
export const WorkspaceLayout = ({ workspaceId, banner, children }: WorkspaceLayoutProps) => {
  return (
    <div>
      ///...
      <div className="relative flex flex-col items-center grow pt-8 md:pt-14 lg:pt-20">{banner}</div>
    </div>
  );
};
```

This way the `WorkspaceLayout is only responsible for layout and we can pop in whatever banner we want.